app/config.py

```diff
@@ -13,6 +13,7 @@ class Config:
     SENTRY_ENABLE = False
     GOOGLE_MAPS_API_KEY = os.environ.get('GOOGLE_MAPS_API_KEY')
 
+    MAIL_LOG_ONLY = os.environ.get('MAIL_LOG_ONLY')
```

Comment:
to be clean, add `False` as second arg (i.e. default)

Reply:
also add `MAIL_LOG_ONLY` to docker-compose.yml so when developing, if someone changes it in `env.py` it actually gets propagated to the app